include/crypto_config_structs.h

```diff
@@ -122,6 +129,7 @@ typedef struct
     TcCheckFecfBool crypto_check_fecf;
     uint8_t vcid_bitmask;
     uint8_t crypto_increment_nontransmitted_iv; // Whether or not CryptoLib increments the non-transmitted portion of the IV field
+    uint8_t padding_control; //Trasmit Padding, or not.
```

Comment:
See my config comment above -- padding_control can be deleted throughout the code.

Reply:
Removed